x/accounts/defaults/lockup/TUTORIAL.md

```diff
@@ -0,0 +1,246 @@
+# Using lockup account on Cosmos sdk
+
+* [Setup](#setup)
+* [Init](#init)
+* [Execution](#execution)
+    * [Delegate](#delegate)
+    * [Undelegate](#undelegate)
+    * [Withdraw reward](#withdraw-reward)
+    * [Withdraw unlocked token](#withdraw-unlocked-token)
+    * [Send coins](#send-coins)
+* [Query](#query)
+    * [Query account info](#query-account-info)
+    * [Query periodic lockup account locking periods](#query-periodic-lockup-account-locking-periods)
+
+## Setup 
+
+To create a lockup account we need 2 wallets (newly created or use any of the existing wallet that you have) one for the granter and one for the owner of the lockup account. 
+
+```bash
+simd keys add granter --keyring-backend test --home ./.testnets/node0/simd/
```

Comment:
can we reword from granter to admin or something like that? granter sounds weird and will confuse people

Reply:
maybe creator would be better

Reply:
+1 and let's drop all the `--home` and `--keyring-backend` flags. Those are redundant, and the not necessary for the tutorial